cypress/support/hub-commands.ts

```diff
@@ -305,3 +305,25 @@ Cypress.Commands.add('createRemoteRegistry', (remoteRegistryName: string) => {
 Cypress.Commands.add('deleteRemoteRegistry', (remoteRegistryId: string) => {
   cy.requestDelete(hubAPI`/_ui/v1/execution-environments/registries/${remoteRegistryId}/`);
 });
+
+Cypress.Commands.add(
+  'deleteCollection',
+  (
+    collectionName: string,
+    namespaceName: string,
+    repository: string,
+    options?: {
+      /** Whether to fail on response codes other than 2xx and 3xx */
+      failOnStatusCode?: boolean;
+    }
+  ) => {
+    cy.requestDelete(
```

Comment:
This is failing in my local computer due to CSRF token missing. But this can be replaced by calling galaxykit command delete collection.